### ISSUE TITLE
Test failure: node_integration_test::test_jpeg

### ISSUE THREAD

**Issue by maaktweluit** [golemfactory/clay#4143]
```
['blender', '-b', '/golem/resources/wlochaty3.blend', '-y', '-P', '/golem/work/render-scripts/scriptfile-crop0_-[crop_num=0].py', '-o', '/golem/output/crop0_', '-noaudio', '-F', 'JPEG', '-t', '7', '-f', '1']
['blender', '-b', '/golem/resources/wlochaty3.blend', '-y', '-P', '/golem/work/render-scripts/scriptfile-crop1_-[crop_num=1].py', '-o', '/golem/output/crop1_', '-noaudio', '-F', 'JPEG', '-t', '7', '-f', '1']
['blender', '-b', '/golem/resources/wlochaty3.blend', '-y', '-P', '/golem/work/render-scripts/scriptfile-crop2_-[crop_num=2].py', '-o', '/golem/output/crop2_', '-noaudio', '-F', 'JPEG', '-t', '7', '-f', '1']
Traceback (most recent call last):
  File "/golem/entrypoints/verifier_entrypoint.py", line 16, in <module>
    params['basefilename'],
  File "/golem/entrypoints/scripts/verifier_tools/verificator.py", line 128, in verify
    make_verdict( subtask_file_paths, crops, results )
  File "/golem/entrypoints/scripts/verifier_tools/verificator.py", line 84, in make_verdict
    metrics_output_filename=os.path.join(OUTPUT_DIR, crop_data['crop']['outfilebasename'] + "metrics.txt"))
  File "/golem/entrypoints/scripts/verifier_tools/img_metrics_calculator.py", line 41, in calculate_metrics
    yres)
  File "/golem/entrypoints/scripts/verifier_tools/img_metrics_calculator.py", line 121, in _load_and_prepare_images_for_comparison
    reference_img = convert_to_png_if_needed(reference_img_path)
  File "/golem/entrypoints/scripts/verifier_tools/img_metrics_calculator.py", line 144, in convert_to_png_if_needed
    return Image.open(file_name)
  File "/usr/local/lib/python3.6/dist-packages/PIL/Image.py", line 2548, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: '/golem/output/crop0_0001.jpeg'
```

It seems that files are saved by blender as `.jpg` instead of the expected `.jpeg`

**Comment by kmazurek** [comment]
Duplicate of: #3989

**Comment by maaktweluit** [comment]
Closing duplicate issue, the other is resolved

**Comment by maaktweluit** [comment]
This problem is back on develop ( and i guess in b0.20 too ): https://buildbot.golem.network/buildbot/#/builders/15/builds/707

**Comment by kmazurek** [comment]
The error reported on the build you linked in caused by the subtasks restart RPC rename (#4264). This should be resolved by #4266.

**Comment by maaktweluit** [comment]
that is 1 of 3 errors reported, check the full log or download the working dirs ( test_jpeg / requestor / golem.error.log )

_edit: wrong test name_

**Comment by kmazurek** [comment]
Ok, got it, let me try reproducing this locally.

**Comment by kmazurek** [comment]
It seems like Buildbot is using an incorrect verifier image tagged as `1.3`. The integration test `test_exr` fails with the following stack trace:
```
Traceback (most recent call last):
  File "/golem/entrypoints/verifier_entrypoint.py", line 15, in <module>
    params['output_format'],
  File "/golem/entrypoints/scripts/verifier_tools/verifier.py", line 221, in verify
    make_verdict(subtask_file_paths, crops, results)
  File "/golem/entrypoints/scripts/verifier_tools/verifier.py", line 132, in make_verdict
    crop_data['crop']['outfilebasename'] + "metrics.txt")
  File "/golem/entrypoints/scripts/verifier_tools/image_metrics_calculator.py", line 63, in calculate_metrics
    PROVIDER_RESULT_CROP_NAME_PREFIX + os.path.basename(reference_crop_path)
  File "/usr/local/lib/python3.6/dist-packages/PIL/Image.py", line 1917, in save
    raise ValueError('unknown file extension: {}'.format(ext))
ValueError: unknown file extension: .exr
```

The lines from the trace are not on Golem's `develop` or any of the release branches. Running `git log -S "PROVIDER_RESULT_CROP_NAME_PREFIX" --all` points to changes from #4203. This would also explain why `test_jpeg` is failing, since the fix from #4176 is not included there. I think the image is a leftover from development and might be used by Buildbot from cache.

**Comment by maaktweluit** [comment]
[Build passed tonight](https://buildbot.golem.network/buildbot/#/builders/15/builds/712), it was a wrongly cached image on buildbot.
This can be caught by the linked issue above, thanks @zakaprov!
